packages/tools/pie-icons-configs/configs.js

```diff
@@ -70,7 +70,7 @@ export function validateGetRegularIconSize (sizeValue) {
  */
 
 export const getSvgProps = (svgClasses, staticClasses, sizeValue, componentName) => {
-    const isLargeIcon = svgClasses.endsWith('Large') || svgClasses.endsWith('-large');
+    const isLargeIcon = svgClasses.endsWith('Large') || svgClasses.endsWith('-large') || componentName.endsWith('Large');
```

Comment:
Do the outputted SVGs for the web component have classnames on them? I'd probably avoid using the componentName, simple as there's an element of user error built in (for instance, we found that some names had lowercase `l`). If you use the generated classnames to check for if the icon is large or not, these have already been normalised against error like that.

Reply:
that makes sense, thank you! I've updated this to use the class [here](https://github.com/justeattakeaway/pie/pull/585/files#diff-beb4382d1fd71394d71c0f0f07f2e5111af3e72f86d94ce723a1a03b48251af9R13) - Closing this current PR as lots of changes had gone through and it was less confusing to start from most recent main 👍